### PR TITLE
Financial Stocks Web Application

### PR DESCRIPTION
**Pull request by d2ndjim**
- Installed working dev dependencies
- Fetched and displayed data from API using redux toolkit react-redux
- Route company stocks to company profiles
- Add responsiveness to web application.
- Live Presentation @ `https://www.loom.com/share/070ba9c986064488a3a0568126e394c7`

### CODE REVIEW

File: src/__tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  expect(loadCompanies()).toEqual(expect.any(Function));
+});
+
+it('loadProfile should return a promise', () => {
+  expect(loadProfile()).toEqual(expect.any(Function));
+});
+
+it('Expect company reducer to handle loadCompanies', () => {
+  expect(companyReducer({}, [loadCompanies.fulfilled])).not.toBeNull();
+});
```
Comment (medaminedev66) [inline src/__tests__/app.test.js:14]: - [ ] Following the project requirements, your app should have unit tests for several components, and also should have integration tests using React Testing Library. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md) to learn about React Testing Library.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @d2ndjim  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your app works as expected and runs without errors.
- Your code organization was perfect.

## Required Changes ♻️
- [ ] Following the project requirements, The video should contain a highlight about a piece of the code you find interesting or are proud of making. Try to open Visual Studio Code or whatever IDE you are using while you are recording a video and try to present a piece of code that interests you.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @d2ndjim,

✔️ Good presentation
✔️ Good design

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

# Status: Approved ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by d2ndjim** [comment]
Thank you.

**Pull request closed without merging**